packages/app/src/debug/DebugPageDetails.vue

```diff
@@ -22,7 +22,7 @@
     v-if="['PASSED', 'OVERLIMIT'].includes(status) || isHidden"
     class="flex flex-col flex-grow w-full p-12 justify-center items-center align-middle "
   >
-    <DebugPassed v-if="status === 'PASSED'" />
+    <DebugPassed v-if="status === 'PASSED' && !isHidden" />
```

Comment:
You could have switched the order with the `DebugOverLimit` below and then added a `v-else-if="status === 'PASSED'"`, but this works just fine.